src/game/game.go

```diff
@@ -0,0 +1,37 @@
+package game
+
+import (
+	"strings"
+	"strconv"
+	"regexp"
+)
+
+func DeserializeBoard(str string) [][]int {
+	var board [][]int
+	// Split string into array based on \n
+	strArr := strings.Fields(str)
+	// Remove first line, last line & middle line which are board's delimiters.
+	strArr = append(strArr[1:4], strArr[5:8]...)
```

Comment:
What does the `...` mean? (I'm curious)

Reply:
append push an item into an array.
To concat 2 arrays you have to push into an array others elements of the second array.
`...` works like JS or Python to spread elements from an array as parameters.

Could be replaced by this : `strArr = append(strArr[1:4], strArr[5], strArr[6], strArr[7], strArr[8])`